packages/client-search/src/types/Log.ts

```diff
@@ -68,4 +68,14 @@ export type Log = {
    * Index name of the log
    */
   readonly index: string;
+  
+  /**
+   * internal queries performed for this query.
```

Comment:
there's no description in the api ref, so I'm not sure if this describes it correctly